package.json

```diff
@@ -99,6 +99,7 @@
     "@testing-library/jest-dom": "^5.11.3",
     "@testing-library/react": "^10.4.3",
     "@testing-library/user-event": "^12.0.14",
+    "@types/react-router-dom": "^5.3.3",
```

Comment:
Adding a new `devDependency` should be noted in the PR, even though it's just a typing library to facilitate developing.

Reply:
thanks.
that will be part of a next improvement of the feature